README.md

```diff
@@ -31,16 +31,19 @@ It should be possible to build everything without Qt Creator, but the guide assu
    Client executables will appear in the `exec` folder, and the server executable will appear in the `griefly-server` folder.  
    Client executables depend from various dlls, so it is needed to manually place them to the `exec` folder.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
+**Note:** It's supposed to be built from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
 
 How to build everything on Linux
 --------------------------------
 
-1. Install dependencies. Look into `.travis.yml` file for clues.
-2. `./make.sh`. Built project will be placed under `exec` directory. Server will be
+1. Install dependencies. Look into `.travis.yml` file for clues. 
+2. `cd` into installation directory.
+3. `git clone https://github.com/griefly/griefly.git`
+4. `cd griefly`
+4. `./make.sh`. Built project will be placed under `exec` directory. Server will be
    built in `gopath/src/griefly-server` directory.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).  
+**Note:** It's supposed to be bult from an active git repository (`git describe --tags --abbrev=4` will be called during the process) -- if you recieve a `fatal: Not a git repository` error, fix with `git init`.
```

Comment:
`git init` does not fix the error, just suppress it. Network protocol relies on git version for compatibility so you will not be able to connect to clients and servers built from actual repository.